forge-gui/res/cardsfolder/upcoming/technomancer.txt

```diff
@@ -0,0 +1,11 @@
+Name:Technomancer
+ManaCost:5 B B
+Types:Artifact Creature Necron Wizard
+PT:5/1
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigMill | TriggerDescription$ When CARDNAME enters the battlefield, mill three cards, then return any number of artifact creature cards with total mana value 6 or less from your graveyard to the battlefield.
+SVar:TrigMill:DB$ Mill | NumCards$ 3 | Defined$ You | SubAbility DBChange
+SVar:DBChange:DB$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | TargetMin$ 0 | TargetMax$ X | MaxTotalTargetCMC$ 6 | ValidTgts$ Artifact.Creature+YouOwn | TgtPrompt$ Select target artifact creature card in your graveyard.
```

Comment:
Doesn't target!

Reply:
Hmm seems like Moorland Rescuer is wrong also ?

Reply:
yea totally